### PR TITLE
Arabic localization

### PR DESCRIPTION
**Pull request by razcodes**
Added arabic support, should work automatically, but if you want to fake the website then add this to your local storage:
`localStorage.setItem("i18nextLng", "ar")`

### CODE REVIEW

File: public/locales/ar/terms.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+    "head":"תנאי השימוש",
```
Comment (omrivardi) [inline public/locales/ar/terms.json:2]: Do we want to translate it?

File: public/locales/he/header.json [hunk 2]
```diff
@@ -8,5 +8,7 @@
   "cleaning": "מנקים את הבית",
   "terms": "תנאי השימוש",
   "contact": "יצירת קשר",
-  "locationAdminLogin": "כניסה לאחראי.ת מוקד"
+  "locationAdminLogin": "כניסה לאחראי.ת מוקד",
```
Comment (omrivardi) [inline public/locales/he/header.json:11]: Need translation?

File: src/components/ProtestForm/ProtestForm.js [hunk 3]
```diff
@@ -28,13 +28,11 @@ const OpeningText = () => {
   return (
     <div style={{ textAlign: 'center' }}>
       <h2>{t('opening.one')}</h2>
-      <h3>{t('opening.two')}</h3>
-      <div>{t('opening.three')}</div>
-      <a href="https://www.youtube.com/watch?v=uhaOAPDAXz4" role="button" tabIndex="1" target="_blank" rel="noopener noreferrer">
+      <p>{t('opening.two')}</p>
+      <p style={{ margin: 0 }}>{t('opening.three')}</p>
+      <a href="#0" role="button" tabIndex="1" onClick={() => window.open('https://www.youtube.com/watch?v=uhaOAPDAXz4')}>
```
Comment (omrivardi) [inline src/components/ProtestForm/ProtestForm.js:33]: Why did you change the link form href to onclick? Also dropping opening.three was on purpose?

**Review by omrivardi — COMMENTED** [review]
Looks good! Some small comments, are translation is still in progress?

Few general notes we need to address:
* The pictures in the popup shows the hebrew website, might need to retake them.
* Do we have a different logo?
* We have data in the location like name, leader etc that won't be translated.
* Need translation for the line of how far are you from the location/

File: public/locales/ar/terms.json (hunk 1 above)
Comment (razcodes) [reply public/locales/ar/terms.json:2]: I have received it like that after translation, I guess there's not literal translation to this in arabic but can recheck

File: public/locales/he/header.json (hunk 2 above)
Comment (razcodes) [reply public/locales/he/header.json:11]: There's a translation here `public/locales/ar/header.json`

File: src/components/ProtestForm/ProtestForm.js (hunk 3 above)
Comment (razcodes) [reply src/components/ProtestForm/ProtestForm.js:33]: This was how the link was implemented in the first place, so I just sticked to it.
Dropping opening.three was on purpose yes.

**Comment by razcodes** [comment]
Need translation for the line of how far are you from the location
-I have tried messing with this and changing the translation but it didn't work out, would be cool if you can take a look and help figure this out :)

**Comment by razcodes** [comment]
Also, as of now the automatic language detection is not working as planned, I am not sure what I did wrong there so maybe you can take a look at that too

File: src/components/ProtestForm/ProtestForm.js (hunk 3 above)
Comment (omrivardi) [reply src/components/ProtestForm/ProtestForm.js]: Sorry, reviewing after a long day :see_no_evil:

**Comment by omrivardi** [comment]
@DanLiebeschutz Do you want to take a look?
Another point which related to localization but not this PR:
Location details are language specific and can't be translated. Possible solution is to filter locations that are written in the same language (add this input to the location form). What do you guys think?

**Comment by DanLiebeschutz** [comment]
Location titles you mean cant be translated? Or the content?

שם אחראי מוקד

או השם עצמו?

File: src/components/ProtestCard/ProtestCard.js [hunk 4]
```diff
@@ -142,10 +141,10 @@ function ProtestCard({ protestInfo, showAction = false, style }) {
         {/* <WazeButton link={`https://www.waze.com/ul?ll=${coordinates?.latitude}%2C${coordinates?.longitude}&navigate=yes&zoom=17`}>
           {t('navigate')}
         </WazeButton> */}
-        <ProtestCardDetail>
+        {/*         <ProtestCardDetail>
```
Comment (omrivardi) [inline src/components/ProtestCard/ProtestCard.js:144]: You can just remove it, better start cleaning the code :)

**Review by omrivardi — APPROVED** [review]
Great!!

**Comment by ofirkg** [comment]
i think there's an issue here
image
